pkgs/tools/misc/wayshot/default.nix

```diff
@@ -1,23 +1,28 @@
-{ lib, fetchFromGitHub, rustPlatform }:
-
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
 rustPlatform.buildRustPackage rec {
   pname = "wayshot";
-  version = "1.3.0";
+  version = "1.3.1";
 
   src = fetchFromGitHub {
     owner = "waycrate";
     repo = pname;
     rev = version;
-    hash = "sha256-WN1qlV6vpIn0uNiE+rXeQTMscNYqkgFytVBc6gJzvyU=";
+    hash = "sha256-nUpIN4WTePtFZTmKAjv0tgj4VTdZeXjoQX6am9+M3ig=";
   };
 
-  cargoHash = "sha256-Hfgr+wWC5zUdHhFMwOBt57h2r94OpdJ1MQpckhYgKQQ=";
+  cargoHash = "sha256-K7E4od+YGb9PgmIzWSQh+sIU8kaJviftNhFpUD9vOIc=";
+
+  # added due to faulty tests in latest version
```

Comment:
Is there an upstream bug report?

Reply:
Did not find any report in source related to this issue. 
In developers makefile they do not run `cargo test`, just `cargo build`. So ignoring checks should not be an issue as they passed it to thier release version.

Reply:
The upstream developers may appreciate a bug report about this, then.

It would also give us something to link to, so that we know what to do about disabling tests in the future. (E.g. if upstream fixes it, we can remove it; if upstream decides they don't care about tests, we will now know that.)

Reply:
Opened [issue](https://github.com/waycrate/wayshot/issues/113). What about pull request to nixpkgs? Should just wait for upstream developers response for the issue or do something else?

Reply:
> What about pull request to nixpkgs? Should just wait for the upstream developers' response to the issue or do something else?

No need to wait, just add a link to the issue in a comment, that explains why tests are disabled.